data/pigui/modules/info-view/01-ship-info.lua

```diff
@@ -56,12 +54,11 @@ local function shipStats()
 			})
 		},
 		false,
-		{ l.WEIGHT_EMPTY..":",  string.format("%dt", player.staticMass - player.usedCapacity) },
-		{ l.CAPACITY_USED..":", string.format("%dt (%dt "..l.FREE..")", player.usedCapacity,  player.freeCapacity) },
-		{ l.CARGO_SPACE..":", string.format("%dt (%dt "..l.MAX..")", player.totalCargo, shipDef.equipSlotCapacity.cargo) },
-		{ l.CARGO_SPACE_USED..":", string.format("%dt (%dt "..l.FREE..")", player.usedCargo, player.totalCargo - player.usedCargo) },
-		{ l.FUEL_WEIGHT..":",   string.format("%dt (%dt "..l.MAX..")", player.fuelMassLeft, shipDef.fuelTankMass ) },
-		{ l.ALL_UP_WEIGHT..":", string.format("%dt", mass_with_fuel ) },
+		{ l.WEIGHT_EMPTY..":",     string.format("%dt", player.staticMass - player.loadedMass) },
```

Comment:
Should use Mass not Weigth..

Reply:
Matches existing behavior - though mass is scientifically correct, weight tends to be used interchangeably in a non-scientific context. Feel free to open a separate PR after merge making changes to this screen.

Reply:
This came from Gliese852 on IRC actually.. I'll defer to them to fix this.

\closing